code/src/index.css

```diff
@@ -1,13 +1,185 @@
+*{
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #191414;
 }
 
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  color: white;
+  font-size: 28px;
+  padding-left: 4%;
+  padding-bottom: 30px;
+  padding-top: 30px;
+}
+
+h2 {
+  color: white;
+  font-size: 14px;
+  font-weight: bold;
+  margin: 5px 0px;
+  max-width: 300px;
+}
+
+h3{
+  color: darkgray;
+  font-size: 14px;
+
+}
+
+.albums-container {
+  display: grid;
+  align-items: end;
+  height: max-content;
+  justify-items: center;
+  align-items: stretch;
+
+}
+
+.image-container {
+  display: flex;
+  position: relative;
+  height: max-content;
+  width: 100%;
+  justify-content: center;
+}
+
+.image-container:hover .album-image {
+ opacity: 0.3;
+}
+
+.image-container:hover .icons-container {
+  opacity: 1;
+ }
+
+.album-image {
+  opacity:1;
+  display: block;
+  margin-bottom: 10px;
+}
+
+.album-card {
+display: grid;
+gap: 10px;
+}
+
+.album{
+  display: flex;
+}
+
+.album-title{
+ margin: 0px;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+  
+}
+
+.artistName {
+  padding-bottom: 20px;
+  display: flex;
+  max-width: 300px;
+}
+
+a:hover{
+  text-decoration: underline white;
+  color: white;
+}
+
+.artist{
+  color: darkgray;
+}
+
+.artist:hover {
+  color: white;
+}
+
+.icons-container{
+  position: absolute;
+  top: 45%;
+  left: 0%;
+  opacity:0;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.icons {
+  width: 8%;
+  filter: invert(1);
+}
+
+.play-icon {
+  width: 12%;
+  transition: transform 0.6s;
+}
+
+.play-icon:hover {
+transform: scale(1.2);
+filter: invert(62%) sepia(24%) saturate(1340%) hue-rotate(88deg) brightness(87%) contrast(96%);
+}
+
+.heart-icon {
+  transition: transform 0.6s;
+}
+
+.heart-icon:hover {
+  transform: scale(1.1);
+  filter: invert(62%) sepia(24%) saturate(1340%) hue-rotate(88deg) brightness(87%) contrast(96%);
+}
+
+.dots-icon {
+  transition: transform 0.6s;
+}
+
+.dots-icon:hover {
+  transform: scale(1.1);
+  filter: invert(62%) sepia(24%) saturate(1340%) hue-rotate(88deg) brightness(87%) contrast(96%);
+}
+
+@media (max-width: 668px){
+  .albums-container {
+    display: grid;
+    grid-template-columns: 2fr;
+    margin: 20px auto;
+  }
```

Comment:
Is this another styling for mobile? What about the styling above?
And secondly, why do you write 2 fr when you want only 1 item?